styles/features.css

```diff
@@ -0,0 +1,49 @@
+.features {
+	background-color: var(--color-secondary);
+}
+
+.features__container {
+	min-height: 600px;
+}
+
+.features__title {
+	text-transform: uppercase;
+	color: var(--color-primary);
+	text-align: center;
+	padding: 50px 0;
+	font-weight: 300;
+}
+
+.features__card {
+	color: var(--color-primary);
+	display: flex;
+	justify-content: center;
+}
+
+.features__card .card {
```

Comment:
W BEM piszemy tylko jedną klasę, jeśli zależy Ci na szczególnych cechach to używasz modyfikatora :)